api/v1alpha3/awscluster_types.go

```diff
@@ -92,6 +92,10 @@ type AWSLoadBalancerSpec struct {
 	// Defaults to false.
 	// +optional
 	CrossZoneLoadBalancing bool `json:"crossZoneLoadBalancing,omitempty"`
+
+	// SecurityGroups sets the security groups used by the load balancer
+	// +optional
+	SecurityGroups []SecurityGroup `json:"securityGroups,omitempty"`
```

Comment:
Do we actually expect users to provide more information here than possibly a SecurityGroup ID? If not, then this should probably use a new type, otherwise users might be confused by what options would be respected/ignored when used in this context.

Reply:
Great call out, how about I make this a `[]string` instead?

Reply:
sgtm, thanks!